<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_edit</class>
 <widget class="QDialog" name="history_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: #202020}

QFrame {
background: #252525 }

QLabel {
color: #c8beb7;
}

QPushButton {
background-color: #182B32;
color: #c8beb7;
}

QComboBox {
background-color: #3e3e3e;
color: #c8beb7;
}

QLineEdit {
background-color: #3e3e3e;
color: #c8beb7;
}

QDateEdit {
background-color: #3e3e3e;
color: #c8beb7;
}

QListWidget { color: #c8beb7; }

QCalendarWidget QWidget { background-color: #c8beb7; }	/* calosc */ 

QCalendarWidget QWidget { alternate-background-color: #c8beb7; } /* naglowek z nazwami dni */

QCalendarWidget QToolButton { color: black; }	/* najwyzszy naglowek - czcionka*/



</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDateEdit" name="date_edit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="line_edit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Edytuj datę:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Edytuj kwotę:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Edytuj kategorię:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="combo_box"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>history_edit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>history_edit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
